plugins/woocommerce-blocks/assets/js/data/checkout/reducers.ts

```diff
@@ -1,12 +1,18 @@
+/**
+ * External dependencies
+ */
+import type { Reducer } from 'redux';
```

Comment:
Should we be importing things straight from redux? I thought the point of wp.data is an abstraction on top, so if they ever want to swap it out for something else it won't break

Reply:
This was removed from the diff after the rebase, but `@wordpress/data` doesn't seem to export any alternative type, for now I think the type def import from redux is fine, as we're not actually using any code.